cmd/serve.go

```diff
@@ -28,32 +28,35 @@ var serveCmd = &cobra.Command{
 	Short: "Start the server",
 	Long:  `Start the go-ship-it server`,
 	Run: func(cmd *cobra.Command, args []string) {
-		lvl := log.INFO
-		switch viper.GetString("server.loglevel") {
-		case "debug":
-			lvl = log.DEBUG
-		case "warn":
-			lvl = log.WARN
-		case "error":
-			lvl = log.ERROR
-		case "off":
-			lvl = log.OFF
-		case "info":
+		logger := logrus.New()
+
+		lvl, err := logrus.ParseLevel(viper.GetString("server.loglevel"))
+		if err != nil {
+			logger.Fatal(err)
+		}
+		logger.SetLevel(lvl)
+
+		switch viper.GetString("server.logformat") {
 		default:
-			lvl = log.INFO
+			logger.Warnf("Could not understand log format '%s'. Defaulting to text", viper.GetString("server.logformat"))
```

Comment:
Might be a little hard to understand that default fall to the next for a default. Consider moving default to the bottom and explicitly set lvl.

Or disregard this comment ;)

Reply:
Noted, and implemented